doc/sphinxext/sphinx_gallery/_static/gallery.css

```diff
@@ -209,7 +209,6 @@ a.sphx-glr-code-links[tooltip]:hover:before{
     display: inline-block;
     text-align: center;
     text-indent: 0;
-    margin-left: -5em;
+    margin-left: 0; /* Has to be zero to avoid overlapping with sidebar */
```

Comment:
OK. I'll have a look once CircleCI is finished running, and we should either consider porting this back to sphinx gallery, or making the CSS change in a scikit-learn specific file.